primefaces/src/main/resources/META-INF/resources/primefaces/locales/locale-en.js

```diff
@@ -39,7 +39,17 @@
          'datatable.sort.DESC': 'activate to sort column descending',
          'datatable.sort.NONE': 'activate to remove sorting on column',
          'columntoggler.CLOSE': 'Close',
-         'overlaypanel.CLOSE': 'Close'
+         'overlaypanel.CLOSE': 'Close',
```

Comment:
@tsotzolas would it be possible for you to translating these `colorpicker.XXX` properties to Greek?

Reply:
Yes of course I will do it !

Reply:
'colorpicker.OPEN': 'Άνοιγμα επιλογέα χρώματος',
'colorpicker.CLOSE': 'Κλείσιμο επιλογέα χρώματος',
'colorpicker.CLEAR': 'Διαγραφή του επιλογέα χρώματος',
'colorpicker.MARKER': 'Κορεσμός: {s}. Φωτεινότητα: {v}.',
'colorpicker.HUESLIDER': 'Ρύθμιση απόχρωσης',
'colorpicker.ALPHASLIDER': 'Ρύθμιση διαφάνειας',
'colorpicker.INPUT': 'Πεδίο τιμής χρώματος',
'colorpicker.FORMAT': 'Μορφή χρώματος',
'colorpicker.SWATCH': 'Δείγμα χρωμάτων',
'colorpicker.INSTRUCTION': 'Επιλογέας κορεσμού και φωτεινότητας. Χρησιμοποιήστε τα πλήκτρα πάνω, κάτω, αριστερά και δεξιά για να επιλέξετε.'

Reply:
Sorry but I don't know where to put it. I hope this is ok.

Reply:
Yes, this will do. Thank you!

Reply:
@tsotzolas do you mind comparing Greek to the locale-en.js English one it looks like there are some othe Greek values missing.  Thanks!